### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Seq/WSeq.lean [hunk 1]
```diff
@@ -1158,7 +1158,6 @@ theorem liftRel_flatten {R : α → β → Prop} {c1 : Computation (WSeq α)} {c
   ⟨S, ⟨c1, c2, rfl, rfl, h⟩, fun {s t} h =>
     match s, t, h with
     | _, _, ⟨c1, c2, rfl, rfl, h⟩ => by
-      -- Porting note: `exists_and_left` should be excluded.
```
Comment (RemyDegenne) [inline Mathlib/Data/Seq/WSeq.lean:1161]: I don't understand what that porting note meant or why we can safely remove it now. Can you explain?

File: Mathlib/Probability/Kernel/Disintegration/CdfToKernel.lean [hunk 2]
```diff
@@ -64,11 +64,11 @@ and for all measurable sets `s : Set β`, `∫ b in s, f (a, b) x ∂(ν a) = (
 Also the `ℚ → ℝ` function `f (a, b)` should satisfy the properties of a Sieltjes function for
 `(ν a)`-almost all `b : β`. -/
 structure IsRatCondKernelCDF (f : α × β → ℚ → ℝ) (κ : kernel α (β × ℝ)) (ν : kernel α β) : Prop :=
-  (measurable : Measurable f)
-  (isRatStieltjesPoint_ae (a : α) : ∀ᵐ b ∂(ν a), IsRatStieltjesPoint f (a, b))
-  (integrable (a : α) (q : ℚ) : Integrable (fun b ↦ f (a, b) q) (ν a))
-  (set_integral (a : α) {s : Set β} (_hs : MeasurableSet s) (q : ℚ) :
-    ∫ b in s, f (a, b) q ∂(ν a) = (κ a (s ×ˢ Iic (q : ℝ))).toReal)
+  measurable : Measurable f
```
Comment (RemyDegenne) [inline Mathlib/Probability/Kernel/Disintegration/CdfToKernel.lean:67]: I had no idea the parentheses around the fields were not needed. That review taught me something. Thanks!

**Review by RemyDegenne — APPROVED** [review]
Thanks!
bors d+

File: Mathlib/Data/Seq/WSeq.lean (hunk 1 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Data/Seq/WSeq.lean:1161]: In mathlib3, it was
```lean
    simp, apply lift_rel_bind _ _ h,
```
which was ported as
```
      -- Porting note: `exists_and_left` should be excluded.
      simp [-exists_and_left]; apply liftRel_bind _ _ h
```
until #11259 squeezed the simp

**Comment by Ruben-VandeVelde** [comment]
Thanks!

bors merge

**Pull request closed without merging**